### DISCUSSION TITLE
Why Supabase doesn't return values from table ?

### DISCUSSION THREAD

**Discussion by Adloya** [supabase/supabase#9965] [Questions] [answered]
Hi,

My code tries to search and extract data from a row that contains a specific email.
The problem is that it returns an empty array.
Here is the code : 
```
const { data, error } = await supabase
        .from('bw_users')
        .select('*')
        .eq('email', user.email)

    if(error) console.log("error : " + error)
    console.log(data)
```

Here is the table
![image](https://user-images.githubusercontent.com/34348778/198905539-f464405c-c6ff-4285-894b-44522085266c.png)

Sorry if i'm dumb, i'm quite new to this.
And sorry for my English, i'm French :/

**Comment by GaryAustin1** [comment, marked as answer]
Usually an empty array on select thru the API means you have RLS enabled and your user does not meet the select policy, or you don't have a policy.

It also is the result if your filters don't match anything.

**Reply by Adloya** [reply]
Thanks !
Didn't check RLS
